releasenotes/notes/fix-sk-for-ugate-0a8c8a99db79c28f.yaml

```diff
@@ -0,0 +1,7 @@
+---
+fixes:
```

Comment:
This does not need a release note since we haven't included SK in a real release yet. Just the RC1 which doesn't publish release notes. So including this would include a fix note for the same release that introduces the feature

Reply:
Ah, we merged it so long ago I forgot it wasn't release yet 😬